packages/opera.rb

```diff
@@ -3,34 +3,35 @@
 class Opera < Package
   description "Opera isn't just a browser. It's an awesome browser."
   homepage 'https://www.opera.com/'
-  version '72.0.3815.400'
+  version '74.0.3911.154'
   compatibility 'x86_64'
-  case ARCH
-  when 'x86_64'
-    source_url 'file:///dev/null'
-    source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  if ARCH == 'x86_64'
+    source_url "https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb"
+    source_sha256 `curl -Ls https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb.sha256sum`.chomp
     depends_on 'alien' => :build
     depends_on 'gtk3'
     depends_on 'gsettings_desktop_schemas'
     depends_on 'harfbuzz'
     depends_on 'graphite'
+    depends_on 'cras'
     depends_on 'sommelier'
   end
 
 
   def self.build
-    system "curl https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb -#o opera.deb"
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera.deb") ) == `curl -Ls https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb.sha256sum`
-    puts "Archive downloaded".lightgreen
-    system "alien -tc opera.deb"
-    system "tar xvf opera-stable-#{version}.tgz > /dev/null"
+    Dir.chdir('../') do
+      system "alien -tc opera-stable_#{version}_amd64.deb"
+      system "tar xf opera-stable-#{version}.tgz"
```

Comment:
We don't need to do this anymore, right?

Reply:
A pr for all deb packages is coming soon:)